app/views/batch_imports/_on_behalf_of.html.erb

```diff
@@ -1,10 +0,0 @@
-<% if Flipflop.proxy_deposit? && current_user.can_make_deposits_for.any? %>
```

Comment:
I presume we cannot use the feature flipper here due to effects elsewhere?

Reply:
@dchandekstark That is correct.  We don't want to disable proxy deposit altogether, just not have this picker widget on this view.